### PR TITLE
Fix Gen.instant when upper and lower bound second part are the same 1.x

### PR DESCRIPTION
**Pull request by strokyl**

### CODE REVIEW

File: test/jvm/src/test/scala/zio/test/TimeVariantsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package zio.test
```
Comment (strokyl) [inline test/jvm/src/test/scala/zio/test/TimeVariantsSpec.scala]: I am pretty sure I am not suppose to put this test here.
Can you tell me where to put it?

Comment (adamgfraser) [reply test/jvm/src/test/scala/zio/test/TimeVariantsSpec.scala]: I think you are looking for `GenSpec.scala`.

**Review by adamgfraser — DISMISSED** [review]
Looks like this just needs a format!

**Comment by strokyl** [comment]
> Looks like this just needs a format!

Done and I almost move my test to GenSpec

**Review by adamgfraser — APPROVED** [review]

**Pull request merged**